File: pgm/ABC_Java/ABC_030_A.java
import java.io.*;
// import java.math.*;
// import java.time.*;
// import java.util.*;
// import java.util.Map.*;
// import java.util.stream.*;

public class Main {
    public static void main (String[] args) throws Exception {
        BufferedReader reader =
            new BufferedReader (new InputStreamReader (System.in));
        final String[] arr = reader.readLine().split (" ");
        final String TAKAHASHI = "TAKAHASHI";
        final String AOKI = "AOKI";
        final String DRAW = "DRAW";
        final int one = 1;
        final int two = 2;
        final int three = 3;
        final int zero = 0;
        final int A = Integer.parseInt (arr[zero]);
        final int B = Integer.parseInt (arr[one]);
        final int C = Integer.parseInt (arr[two]);
        final int D = Integer.parseInt (arr[three]);
        System.out.println (B * C > D * A   ? TAKAHASHI
                            : B * C < D * A ? AOKI
                                            : DRAW);
        System.exit (0);
    }
}

File: pgm/ABC_R/ABC_030_A.r
con <- file(description = "stdin", open = "r")
arr <- readLines(con = con)
close(con)
TAKAHASHI <- "TAKAHASHI"
AOKI <- "AOKI"
DRAW <- "DRAW"
blank <- ""
space <- " "
one <- 1L
two <- 2L
three <- 3L
four <- 4L
A <- strsplit(x = arr[one], split = space, fixed = TRUE)[[one]][one] |>
  as.integer()
B <- strsplit(x = arr[one], split = space, fixed = TRUE)[[one]][two] |>
  as.integer()
C <- strsplit(x = arr[one], split = space, fixed = TRUE)[[one]][three] |>
  as.integer()
D <- strsplit(x = arr[one], split = space, fixed = TRUE)[[one]][four] |>
  as.integer()
if (B * C > A * D) {
  TAKAHASHI |>
    cat("\n", sep = blank)
} else if (B * C < A * D) {
  AOKI |>
    cat("\n", sep = blank)
} else {
  DRAW |>
    cat("\n", sep = blank)
}
q("no")